Bricks/net/tcp/test.cc

```diff
@@ -240,13 +241,17 @@ TEST(TCPTest, EchoLongMessageTestsDynamicBufferGrowth) {
 }
 
 // Don't run this ~1s test more than once in a loop.
-struct OnlyCheckForUnresolvableURLOnceSingleton {
+struct RunResolveAddressTestOnlyOnceSingleton {
   bool done = false;
 };
 
-TEST(TCPTest, ResolveAddressException) {
-  bool& done = Singleton<OnlyCheckForUnresolvableURLOnceSingleton>().done;
+TEST(TCPTest, ResolveAddress) {
+  bool& done = Singleton<RunResolveAddressTestOnlyOnceSingleton>().done;
   if (!done) {
+    EXPECT_EQ("127.0.0.1", ResolveIPFromHostName("localhost"));
+    EXPECT_EQ("127.0.0.1", ResolveIPFromHostName("127.0.0.1"));
+    EXPECT_EQ("8.8.8.8", ResolveIPFromHostName("8.8.8.8"));
```

Comment:
I know what 8.8.8.8 is and it confuses me. Either a comment "does not require Internet connection", and/or changing the IP to some `101.102.103.104` is in order, IMHO.